manifests/alertmanager.pp

```diff
@@ -107,23 +107,23 @@
   Array $templates,
   String $user,
   String $version,
-  Boolean $service_enable        = true,
-  String $service_ensure         = 'running',
-  String $service_name           = 'alertmanager',
-  Boolean $restart_on_change     = true,
-  Boolean $purge_config_dir      = true,
-  Boolean $manage_config         = true,
-  String $init_style             = $prometheus::init_style,
-  String $install_method         = $prometheus::install_method,
-  Boolean $manage_group          = true,
-  Boolean $manage_service        = true,
-  Boolean $manage_user           = true,
-  String $os                     = $prometheus::os,
-  String $extra_options          = '',
-  Optional[String] $download_url = undef,
-  String $config_mode            = $prometheus::config_mode,
-  String $arch                   = $prometheus::real_arch,
-  Stdlib::Absolutepath $bin_dir  = $prometheus::bin_dir,
+  Boolean $service_enable           = true,
+  String[1] $service_ensure         = 'running',
```

Comment:
Unrelated so probably better for a different PR, but there's a `Stdlib::Ensure::Service` type

Reply:
thanks, will do this in another PR